### PR TITLE
Create scaffolder template to construct Roadie plugins monorepo 

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

File: scaffolder-templates/roadie-plugin/template.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      name: Fill in names and descriptions
+      action: fs:rename
+      input:
+        files:
+          - from: ./plugins/my-plugin
+            to: ./plugins/${{ parameters.plugin_name }}
+
+    - id: publish
+      name: Publish
+      action: publish:github
```
Comment (iain-b) [inline scaffolder-templates/roadie-plugin/template.yaml:54]: I was thinking it'd be cool to allow users to create new plugins via the scaffolder too. I think that'd need/be best put in a new (similar) template? clearly doesn't block this work.

**Review by iain-b — APPROVED** [review]
just one comment

File: scaffolder-templates/roadie-plugin/template.yaml (hunk 1 above)
Comment (Xantier) [reply scaffolder-templates/roadie-plugin/template.yaml:54]: Oh, that would be nice. I was also thinking of creating another one where they can scaffolder the internal plugin itself into an existing monorepo, but it's probably more easily achieved by copy-pasting existing code.


Adding new plugin would be something like:
* Create this plugin skeleton
* Run GitHub action that bundles the plugin and ships it to (where? Where to get creds?)
* Register plugin to Roadie UI

Or did you have something else in mind?

Comment (Xantier) [reply scaffolder-templates/roadie-plugin/template.yaml:54]: Or possible a set of Localhost serving plugin components initially so they can start developing

**Pull request merged**